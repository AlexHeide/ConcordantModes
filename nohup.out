Bond s-vectors:
[  9.98529915e-01   5.42034069e-02  -2.72721638e-07]
[ -9.98529915e-01  -5.42034069e-02   2.72721638e-07]
[ 0.  0.  0.]
[ -3.41810175e-01   9.39768909e-01  -4.50017750e-04]
[  3.41810175e-01  -9.39768909e-01   4.50017750e-04]
[ 0.  0.  0.]
[-0.35526773 -0.46090983  0.81323242]
[ 0.35526773  0.46090983 -0.81323242]
[ 0.  0.  0.]
[-0.35522836 -0.46164323 -0.81283352]
[ 0.35522836  0.46164323  0.81283352]
[ 0.  0.  0.]
[  3.53184719e-01  -9.35553609e-01  -8.42003585e-06]
[ -3.53184719e-01   9.35553609e-01   8.42003585e-06]
[ 0.  0.  0.]
Angle s-vectors:
[ -4.58198613e-01  -1.66654713e-01   6.67182687e-05]
[  4.37948695e-01   5.39696746e-01  -2.42441648e-04]
[  2.02499182e-02  -3.73042033e-01   1.75723380e-04]
[ 0. -0.  0.]
[-0.45274974  0.06333766 -0.16189027]
[ 0.46240255 -0.24115929  0.49030577]
[-0.00965282  0.17782163 -0.3284155 ]
[ 0.  0. -0.]
[-0.45275758  0.06348366  0.16181115]
[ 0.46242629 -0.24160147 -0.49006565]
[-0.00966872  0.1781178   0.32825449]
[ 0.  0.  0.]
[  5.16889431e-01   1.95133070e-01   1.31543050e-06]
[ -4.96639510e-01  -5.68175144e-01  -4.58283291e-06]
[ -2.02499204e-02   3.73042074e-01   3.26740241e-06]
[ 0.  0.  0.]
Torsion s-vectors:
[-0.0249796   0.46017306  0.24989638]
[ 0.03230224 -0.59507343 -0.94587639]
[-0.00730978  0.13466106  0.18646222]
[ -1.28515362e-05   2.39313368e-04   5.09517788e-01]
[-0.  0.  0.]
[ 0.02496749 -0.45994745  0.25031259]
[-0.03225516  0.59419671 -0.94641435]
[ 0.00730052 -0.13448858  0.18658397]
[ -1.28515362e-05   2.39313368e-04   5.09517788e-01]
[-0.  0.  0.]
[ -4.32562830e-07   5.05266540e-06  -5.79547880e-01]
[  3.38024661e-06  -5.93296158e-05   5.84515596e-01]
[ -1.57992200e-05   2.93590319e-04   5.04550073e-01]
[  1.28515362e-05  -2.39313368e-04  -5.09517788e-01]
[-0.  0.  0.]
