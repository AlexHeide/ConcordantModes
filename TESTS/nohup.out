
General::obspkg: 
   VectorAnalysis` is now obsolete. The legacy version being loaded may
     conflict with current functionality. See the Compatibility Guide for
     updating information.

General::compat: 
   Combinatorica Graph and Permutations functionality has been superseded by
    preloaded functionality. The package now being loaded may conflict with
    this. Please see the Compatibility Guide for details.
{1.7961501993211249, 1.4312592441883532, 2.5636034348214257*^-19}
{-7.887512865067947*^-11, -9.701164693787234*^-8, 2.5636034348214257*^-19}
Without symmetry 7 displaced geometries are required

With symmetry 7 displaced geometries are required

Your job-array 337969.1-7:1 ("MixedHess") has been submitted
Job 337969.1 exited with exit code 0.
Job 337969.2 exited with exit code 0.
Job 337969.3 exited with exit code 0.
Job 337969.4 exited with exit code 0.
Job 337969.5 exited with exit code 0.
Job 337969.6 exited with exit code 0.
Job 337969.7 exited with exit code 0.

General::obspkg: 
   VectorAnalysis` is now obsolete. The legacy version being loaded may
     conflict with current functionality. See the Compatibility Guide for
     updating information.

General::compat: 
   Combinatorica Graph and Permutations functionality has been superseded by
    preloaded functionality. The package now being loaded may conflict with
    this. Please see the Compatibility Guide for details.
The energies at the displaced geometries:
MatrixForm[{{-76.34578231037413, -76.34578030045076, -76.34578029457806, -76.34575820213925, -76.3457578019832, -76.34568302318587, -76.3456830231861}}]
  basis=cc-pVTZ

['H', 'O', 'H']
['R', 'R']
[['2', '1'], ['3', '2']]
['A']
[['3', '2', '1']]
['']
[['']]
{'A': 1.8103975742886766, 'R': 1.810017290509182}
['H', 'O', 'H']
[['2', '1'], ['3', '2']]
['R', 'R']
[['3', '2', '1']]
['A']
[]
[]
{'A': '1.8103975742886766', 'R': '1.810017290509182'}
This program took 114.39802503585815 seconds to run.
